docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -0,0 +1,494 @@
+
+<AuthzModelSnippetViewer
+  configuration={{
+    type_definitions: [
+      { type: 'user', relations: {} },
+      {
+        type: 'document',
+        relations: { viewer: { this: {} } },
+        metadata: {
+          relations: {
+            viewer: {
+              directly_related_user_types: [{ type: 'user' }, { type: 'user', wildcard: {} }],
+            },
+          },
+        },
+      },
+    ],
+    schema_version: '1.1',
+  }}
+/>
+
+You’ll need to specify `user:*` as the user value in the tuple to enable this:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All objects of type "user" are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+Being explicit about the wildcard type restrictions also lets you model scenarios like “all employees can see this document, but not all external users”, “all user accounts can access this document, but not service/machine-to-machine accounts”.
+
+This change implies that you’ll need to change your code to write tuples with this new syntax, and that you’ll need to migrate existing tuples to use the new format.
+
+You might have 3 kinds of tuples in your model that use “\*”, with different migration strategies:
+
+1. Tuples that have user = “\*”
+
+You would need to retrieve those tuples and write them using the proper type (e.g. `user:*`). To retrieve them, you’ll need to use the Read endpoint, filter on your side the tuples that have `user = “*”`, and call the Write API for each one, with the proper type, e.g:
+
+<RelationshipTuplesViewer
+  relationshipTuples={[
+    {
+      user: 'user:*',
+      relation: 'viewer',
+      object: 'document:pricing',
+      _description: 'All objects of type "user" are viewers of the "pricing" document',
+    },
+  ]}
+/>
+
+2. Tuples that have `user = "employee:*”`, where `employee` is NOT a type that is defined in the new iteration of your model.
+
+If you have tuples with this format, they will be considered invalid because they don’t have a corresponding type in the model. If you need such a type defined, you’ll need to add it to the model, and the scenario will be similar to the one described below.
+
+3. Tuples that have `user = “user:*”`, which would mean "the user with user_id = '\*'”, where `user` is type that is defined in the new iteration of your model.
+
+In this case, the meaning of the tuple will change. If you were intending to specify "a user with user id = \*", you will need to encode it in a different way instead of using “\*”. If you intended to specify “every user has this relationship with this object” then it’s not the way it would have worked with schema version = 1.0, but it will work with version = 1.1.
```

Comment:
> If you intended to specify “every user has this relationship with this object” then it’s not the way it would have worked with schema version = 1.0, but it will work with version = 1.1.

This isn't quite clear. I think it may make more sense to say

"If you intended to specify public access to the object (e.g. "every user has this relationship with this object"), 1.0 models will not interpret the `user:*` value as a wildcard but 1.1 models will.